apps/studio/components/interfaces/Storage/AnalyticsBuckets/index.tsx

```diff
@@ -111,15 +105,20 @@ export const AnalyticsBuckets = () => {
                     </TableRow>
                   )}
                   {analyticsBuckets.map((bucket) => (
-                    <TableRow key={bucket.id}>
+                    <TableRow
+                      key={bucket.id}
+                      className="cursor-pointer h-16"
```

Comment:
The default height of these rows felt a bit puny for now-tappable elements. Is `h-16` the best way to do this? Padding wouldn’t cooperate.